pkg/cryptoauthlib/doc.txt

```diff
@@ -23,6 +23,15 @@
  * USEPKG += cryptoauthlib
  * to your Makefile.
  *
+ *
+ * ### Shell
+ *
+ * To facilitate the device configuration the RIOT shell provides some
+ * helper commands. Currently there are shell commands to read the device's
+ * config zone, to permanently lock the config zone and to check whether config
+ * and data zone are locked.
+ *
```

Comment:
Now that you have this nice piece of documentation, how about you mention how to enable the shell command? (`USEPKG += cryptoauthlib`)

Reply:
In case there are "dangerous"shell commands that constantly lock zones, you could mention it here too.

Reply:
I  just realized that `USEPKG += cryptoauthlib` was already present. Anyway, the additional text doesn't hurt but needs clarification. The shell handler is enabled by **including** the package in the Makefile of an application that also includes the shell (i.e. examples/default).